.pre-commit-config.yaml

```diff
@@ -0,0 +1,11 @@
+repos:
+  - repo: https://github.com/psf/black
+    rev: 22.8.0
+    hooks:
+      - id: black
+        language_version: python3.6
+  - repo: https://github.com/pycqa/isort
+    rev: 5.10.1
+    hooks:
+      - id: isort
+        args: ["--profile", "black"]
```

Comment:
end-of-file

Reply:
maybe add the yaml block to pre-commit 🤔

Reply:
Added yaml, eof, and trailing whitespace checks